### ISSUE TITLE
Resolution selector instead of scaling

### ISSUE THREAD

**Issue by Zorakie** [QuanTrieuPCYT/Dell-Latitude-5480_Hackintosh#6]
Hello,
after installing Monterey with this EFI on e5480 (actually with using a suggested tweak for a graphics adapter in this issue thread https://github.com/QuanTrieuPCYT/Dell-Latitude-5480_Hackintosh/issues/4 ), I run to yet another issue with graphics settings. For some reason, macOS detects the built-in display in a weird way.
All reports fine, in About this Mac, it reports as built in display, correct resolution, correct size (13,9-inch), but in Settings->Display, the system obviously things it is an external monitor as I understood from reading and searching online, as for built-in displays, in the scaling option, there should be a selector for text size. However, my e5480 shows resolution instead, which should be available for external displays.

This is a problem, as on fullHD 13,9-inch display, everything is just way too small. And changing the resolution makes things blurry (who would guess). External utilities like RDM does not really work as they also work with resolution rather than with text (UI) scale. I also tried "defaults write /Library/Preferences/com.apple.windowserver.plist DisplayResolutionEnabled -bool false|true" from terminal (sudo, then restarts), still didn't work. As suggested, also tried to click on Scaled radio-button with key modifiers (ctrl, command, shift, alt), also with no results.

As a last resort, I tried also to do a complete reinstall using the EFI built (because the original installation was still using the wrong display setup in EFI and the EFI was tweaked while the system was already installed, so who knows, what could be loaded wrongly to the OS) - but yet, no change, still tucked on resolutions rather than scaling.

Checked with original macBook Air of my friend, laptos do have text scaling rather than resolutions.

Any idea, how to fix that?

So basically:
This is what I see:
![obrazek](https://user-images.githubusercontent.com/14437576/163624701-9abcdf27-6243-445d-834c-6942723aead8.png)

And this is what I need:
![obrazek](https://user-images.githubusercontent.com/14437576/163624735-a2af77b2-845b-4531-83ba-f75ef3e4c9c3.png)

**Comment by QuanTrieuPCYT** [comment]
https://github.com/xzhih/one-key-hidpi

I used this to get HiDPI support for the built-in display on my Dell, you should also try that lol
Will add this thing into the README
